Assets/Scripts/Player.cs

```diff
@@ -35,6 +36,11 @@ private void Update()
         MovementControl();
         CheckStatus();
     }
+    
+    public float GetPityTime()
```

Comment:
În C# se folosesc properties în loc de a scrie getteri și setteri manual ca în Java.

https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/using-properties